app/Models/Task.php

```diff
@@ -3,14 +3,17 @@
 declare(strict_types=1);
 
 namespace App\Models;
-
-class Items
+use Exception;
+class Task
 {
-    public function addItems(): void
+    public function addTask(): void
     {
         if (file_exists('data.json')) {
```

Comment:
you use 'data.json' a lot, mb it would be more dynamic to go for const then, so if file is changed you won't have to do it for all inputs